internal/sql/sqlpath/read.go

```diff
@@ -46,7 +46,7 @@ func Glob(patterns []string) ([]string, error) {
 				files = append(files, filepath.Join(path, f.Name()))
 			}
 		} else {
-			files = append(files, path)
+			files = append(files, filepath.Clean(path))
```

Comment:
The result of `filepath.Join` is `Clean`'d, so these two branches didn't have consistent behavior before